stable/redis-ha/templates/redis-ha-statefulset.yaml

```diff
@@ -91,21 +96,31 @@ spec:
         volumeMounts:
         - mountPath: /data
           name: data
+        - mountPath: /probes
+          name: probes
       - name: sentinel
         image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
         imagePullPolicy: {{ .Values.image.pullPolicy }}
         command:
           - redis-sentinel
         args:
           - /data/conf/sentinel.conf
+{{- if .Values.auth }}
+        env:
+        - name: AUTH
+          valueFrom:
+            secretKeyRef:
+              name: {{ template "redis-ha.fullname" . }}
+              key: auth
+{{- end }}
         livenessProbe:
           exec:
-            command: ["redis-cli", "-p", "{{ .Values.sentinel.port }}", "ping"]
+            command: [ "sh", "/probes/check-quorum.sh", "{{ .Values.sentinel.port }}"]
           initialDelaySeconds: 15
```

Comment:
It will take at least 60 seconds (15s readiness + 3 * 15s liveness failures) for a pod incorrectly initialized as a master to realize it and restart. Seems like a long time.

Could we not check for quorum/slave count in the readiness probe as well to reduce this time to 15 seconds?

Reply:
Yeah I agree it could still take too long. I'll change it and test it out. 

Regardless, I have another commit to expose all the liveness/readiness settings in the values.yaml but I left it out of this PR as it made it drastically larger because of the corresponding README changes. I'll open up another PR after this one just for those changes.